app/controllers/users_controller.rb

```diff
@@ -16,20 +16,28 @@ def show
   # GET /user/new
   def new
     @user = User.new(role: "Player")
+    @teams = Team.all
+    @team_roles = TeamRole.all
+    @games = Game.all
   end
 
   # GET /user/1/edit
   def edit
+    @teams = Team.all
+    @team_roles = TeamRole.all
   end
 
   # POST /user
   # POST /user.json
   def create
     @user = User.new(user_params)
+    puts "NEW USER"
+    puts @user
```

Comment:
This was me testing and should be removed, but FYI it never seems to actually get to this point.